README.md

```diff
@@ -97,7 +97,33 @@ Many browsers have started blocking cross-origin or "third party" cookies by def
   * [token.getWithoutPrompt](#tokengetwithoutpromptoptions) must have access to cookies on the Okta domain via an iFrame running on your application's page.
   * [token.renew](#tokenrenewtokentorenew) uses [token.getWithoutPrompt](#tokengetwithoutpromptoptions) and is subject to the same limitations.
 
-If your application depends on any of these methods, you should try to either rewrite your application to avoid using these methods or communicate to your users that they must enable third party cookies. Okta engineers are currently working on a better long-term solution to this problem.
+Notably, a browser blocking 3rd party cookies prevents automatic token renewal from working.   The code will behave as if the user's session at the issuer site has expired.
+
+#### How to handle Third party cookie issues
+
+Note that third party cookie blocking is controlled by the browser.  `okta-auth-js` calls run in the browser can be affected, but `okta-auth-js` calls made on the server cannot.  These solutions assume you are running `okta-auth-js` from the browser.
+
+##### Solution 1: Use a custom domain URL  
+  * If the domain of your app and the domain that holds the session cookie are the same domain, there is no "third" party to trigger issues with third-party cookies
+  * You can learn more about how to use this feature here: https://developer.okta.com/docs/guides/custom-url-domain/overview/
+  * After making those changes, you will need to make sure your app is configured to use the matching url
+  * This should prevent third-party cookie issues
+
```

Comment:
There is another possible solution: self host the signin widget and do not use redirect. This can be done by calling `getWithoutPrompt` with the sessionToken. https://github.com/okta/samples-js-react/pull/137

We are planning to make this flow better supported within the widget via a config option so that nobody needs to use the sessionToken directly, but this is a valid workaround today.